pyproject.toml

```diff
@@ -43,6 +43,7 @@ medcat = {version = ">=1.2.6", optional = true}
 anndata = ">=0.7.8"
 fancyimpute = "^0.7.0"
```

Comment:
Could you please for all of them use >= and not ^?

Reply:
no problem